package/run.ps1

```diff
@@ -46,6 +46,22 @@ function Get-StringHash {
     return (Get-FileHash -InputStream $stringAsStream -Algorithm SHA256).Hash.ToLower()
 }
 
+# stops the RKE2 process as well as other processes spawned from rke2
+function Stop-RKE2-Processes () {
+    $ProcessNames = @('rke2', 'kube-proxy', 'kubelet', 'containerd', 'calico-node')
+    foreach ($ProcessName in $ProcessNames) {
+        Write-LogInfo "Checking if $ProcessName process exists"
+        if ((Get-Process -Name $ProcessName -ErrorAction SilentlyContinue)) {
+            Write-LogInfo "$ProcessName process found, stopping now"
+            Stop-Process -Name $ProcessName
+            while (-Not(Get-Process -Name $ProcessName).HasExited) {
```

Comment:
nit: spacing, it would be ideal if we can do something like below unless it's a powershell limitation.

```suggestion
            while (-Not (Get-Process -Name $ProcessName).HasExited) {
```